scripts/redeploy.py

```diff
@@ -2,7 +2,7 @@
 #
 # auto-deploy script for https://riot.im/develop
 #
-# Listens for HTTP hits. When it gets one, downloads the artifact from jenkins
+# Listens for HTTP hits. When it gets one, downloads the artifact from buildkite
```

Comment:
Maybe link to  https://buildkite.com/docs/apis/webhooks somewhere in the file.

Reply:
done